evm/utils.js

```diff
@@ -200,6 +200,20 @@ const isString = (arg) => {
     return typeof arg === 'string';
 };
 
+const isStringArray = (arr) => {
+    if (!Array.isArray(arr)) {
+        return false;
+    }
+
+    for (const item of arr) {
+        if (!isString(item)) {
+            return false;
+        }
+    }
+
+    return true;
+};
```

Comment:
```suggestion
const isStringArray = (arr) => Array.isArray(arr) && arr.every(isString);
```
This should be a one-liner.